web_external/js/views/body/MapPanel.js

```diff
@@ -214,9 +215,10 @@ minerva.views.MapPanel = minerva.View.extend({
                     }
                 })
             });
-            this.map.createLayer(this.session.sessionJsonContents.basemap);
+            this.map.createLayer(this.session.sessionJsonContents.basemap,
```

Comment:
This is essentially what my comment on #225 addresses. I'll change it if you want, but I'm only doing retrieval here and it is identical to how retrieval of session data is done throughout the codebase.